rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/ExportToCatalogConfirmWindow.java

```diff
@@ -0,0 +1,356 @@
+ */
+package org.ow2.proactive_grid_cloud_portal.rm.client.nodesource.serialization;
+
+import static org.ow2.proactive_grid_cloud_portal.rm.shared.CatalogConstants.EXPORT_FAILED_MESSAGE;
+import static org.ow2.proactive_grid_cloud_portal.rm.shared.CatalogConstants.INITIAL_COMMIT_MESSAGE;
+import static org.ow2.proactive_grid_cloud_portal.rm.shared.CatalogConstants.NODE_SOURCE_CONTENT_TYPE;
+import static org.ow2.proactive_grid_cloud_portal.rm.shared.CatalogConstants.NODE_SOURCE_KIND;
+
+import org.ow2.proactive_grid_cloud_portal.common.client.Images;
+import org.ow2.proactive_grid_cloud_portal.common.client.json.JSONUtils;
+import org.ow2.proactive_grid_cloud_portal.common.client.model.LoginModel;
+import org.ow2.proactive_grid_cloud_portal.rm.client.NodeSourceConfiguration;
+import org.ow2.proactive_grid_cloud_portal.rm.client.RMController;
+import org.ow2.proactive_grid_cloud_portal.rm.shared.CatalogConstants;
+import org.ow2.proactive_grid_cloud_portal.rm.shared.ServletMappings;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.http.client.Request;
+import com.google.gwt.http.client.RequestBuilder;
+import com.google.gwt.http.client.RequestCallback;
+import com.google.gwt.http.client.RequestException;
+import com.google.gwt.http.client.Response;
+import com.google.gwt.json.client.JSONArray;
+import com.google.gwt.json.client.JSONObject;
+import com.google.gwt.json.client.JSONParser;
+import com.google.gwt.user.client.rpc.AsyncCallback;
+import com.google.gwt.user.client.ui.FormPanel;
+import com.google.gwt.user.client.ui.HasVerticalAlignment;
+import com.google.gwt.user.client.ui.Hidden;
+import com.google.gwt.user.client.ui.HorizontalPanel;
+import com.google.gwt.user.client.ui.ListBox;
+import com.google.gwt.user.client.ui.TextBox;
+import com.google.gwt.user.client.ui.VerticalPanel;
+import com.smartgwt.client.types.Alignment;
+import com.smartgwt.client.types.VerticalAlignment;
+import com.smartgwt.client.util.SC;
+import com.smartgwt.client.widgets.IButton;
+import com.smartgwt.client.widgets.Label;
+import com.smartgwt.client.widgets.Window;
+import com.smartgwt.client.widgets.layout.HLayout;
+import com.smartgwt.client.widgets.layout.VLayout;
+
+
+public class ExportToCatalogConfirmWindow extends Window {
+
+    public static final String WINDOW_TITLE = "Export Node Source to Catalog";
+
+    public static final int WINDOW_WIDTH = 680;
+
+    public static final int WINDOW_HEIGHT = 200;
+
+    public static final String SELECT_A_BUCKET_OPTION = "Select a bucket";
+
+    private String nodeSourceName;
+
+    private RMController rmController;
+
+    private NodeSourceConfigurationParser parser;
+
+    private ExportToCatalogHiddenPanel hiddenFormItemsPanel;
+
+    private NodeSourceSerializationFormPanel exportNodeSourceToCatalogForm;
+
+    private Label windowLabel;
+
+    private ListBox bucketList;
+
+    private TextBox commitMessage;
+
+    private Label revisionLabel;
+
+    private boolean isRevision;
```

Comment:
[here](https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter) they advocate that boolean member should not start with **is**.

Reply:
I kind of agree. I changed it into "revised".